channeldb/invoice_test.go

```diff
@@ -1090,6 +1090,8 @@ func TestQueryInvoices(t *testing.T) {
 	for i := 1; i <= numInvoices; i++ {
 		amt := lnwire.MilliSatoshi(i)
 		invoice, err := randInvoice(amt)
+		invoice.CreationDate = invoice.CreationDate.Add(
+			time.Duration((i - 1)) * time.Second)
```

Comment:
nit: formatting and superfluous inner parenthesis.

Reply:
hawkeye! fixed!